modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -78,64 +78,89 @@ const assertStructureHasCustomStyle = (editor: Editor, rows: number, columns: nu
+  }
   await TinyUiActions.pWaitForUi(editor, `div[title="${itemTitle.getOr('None')}"]`);
   TinyUiActions.clickOnUi(editor, `div[title="${itemTitle.getOr('None')}"]`);
   closeMenu(editor);
 };
 
-const pAssertNoCheckmarksInMenu = async (editor: Editor, menuTitle: string, expectedFalseCheckmarks: number, container: SugarElement<HTMLElement>) => {
+const pAssertNoCheckmarksInMenu = async (editor: Editor, menuTitle: string, expectedFalseCheckmarks: number, container: SugarElement<HTMLElement>, toolbar: boolean) => {
   const expected = {
-    '.tox-menu': 1,
+    '.tox-menu': toolbar ? 1 : 2,
     '.tox-collection__item[aria-checked="true"]': 0,
     '.tox-collection__item[aria-checked="false"]': expectedFalseCheckmarks,
   };
 
-  await pAssertMenuPresence(editor, 'Menu should open, but not have any checkmarks', menuTitle, expected, container);
+  await pAssertMenuPresence(editor, 'Menu should open, but not have any checkmarks', menuTitle, expected, container, toolbar);
 };
 
-const pAssertStyleCanBeToggledOnAndOff = async (editor: Editor, options: AssertStyleOptions) => {
+const pAssertStyleCanBeToggled = async (editor: Editor, options: AssertStyleOptions, toolbar: boolean) => {
   const sugarContainer = SugarBody.body();
   setEditorContentTableAndSelection(editor, options.rows, options.columns);
-  await pAssertNoCheckmarksInMenu(editor, options.menuTitle, options.checkMarkEntries, sugarContainer);
+  await pAssertNoCheckmarksInMenu(editor, options.menuTitle, options.checkMarkEntries, sugarContainer, toolbar);
 
-  await pClickOnSubMenu(editor, options.menuTitle, Optional.some(options.subMenuTitle));
-  await pAssertCheckmarkOn(editor, options.menuTitle, options.subMenuTitle, options.checkMarkEntries - 1, sugarContainer);
+  await pClickOnSubMenu(editor, options.menuTitle, Optional.some(options.subMenuTitle), toolbar);
+  await pAssertCheckmarkOn(editor, options.menuTitle, options.subMenuTitle, options.checkMarkEntries - 1, sugarContainer, toolbar);
   assertStructureHasCustomStyle(editor, options.rows, options.columns, options.customStyle);
 
-  await pClickOnSubMenu(editor, options.menuTitle, Optional.none());
-  await pAssertNoCheckmarksInMenu(editor, options.menuTitle, options.checkMarkEntries, sugarContainer);
+  await pClickOnSubMenu(editor, options.menuTitle, Optional.none(), toolbar);
+  await pAssertNoCheckmarksInMenu(editor, options.menuTitle, options.checkMarkEntries, sugarContainer, toolbar);
   assertStructureIsRestoredToDefault(editor, options.rows, options.columns);
 };
 
+const pAssertStyleCanBeToggledOnAndOff = async (editor: Editor, options: AssertStyleOptions) => {
+  await pAssertStyleCanBeToggled(editor, options, true);
+  await pAssertStyleCanBeToggled(editor, options, false);
+};
+
 export {
   pAssertStyleCanBeToggledOnAndOff,
   setEditorContentTableAndSelection,
   pAssertNoCheckmarksInMenu,
-  pAssertMenuPresence
+  pAssertMenuPresence,
+  clickOnButton,
+  pClickOnMenuItem as clickOnMenuItem,
```

Comment:
This wasn't entirely correctly renamed. It was only renamed internally, so please make sure to remove the alias and rename it outside this module as well.

Reply:
Dammit.